quartz/plugins/transformers/ofm.ts

```diff
@@ -344,6 +348,28 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
             })
           }
 
+          if (opts.enableYouTubeEmbed) {
+            visit(tree, "image", (node: any) => {
```

Comment:
let's push to the regex replacement array (also avoid typing things with `any`)

docs: https://github.com/syntax-tree/mdast-util-find-and-replace

Reply:
it seems `findAndReplace` only works on [text nodes](https://github.com/syntax-tree/mdast-util-find-and-replace/blob/main/lib/index.js#L90), i wasn't able to process image nodes with it.

i was also having a hard time with the typing, think it's because i'm trying to convert an image node to an html node, so i moved the conversion to run as an html plugin.